### ISSUE TITLE
Converting form MATLAB syntax to python syntax

### ISSUE THREAD

**Issue by emorevival** [Notebook-Factory/RRIFT_notebooks#2]
Hi, I'm having trouble trying to recreate something in python that the authors are doing in MATLAB. This is the line I'm having trouble with:
`plot(t, Ct(:,idx), 'LineWidth', 1)`. I can see that the x parameter of the plot is `t`, and the y parameter is `Ct(:,idx)`. After some googling I found out that array(:,n) will only return the n-th column, but the problem is that `idx` is not a number, but a 1d array. Seeing as `idx` has 4 elements and the figure has 4 plots, we thought that maybe it takes the i-th element for every plot. I tried to get the 5th column of `Ct` (5 is the first element) but I got an incorrect plot: ![incorrect plot](https://user-images.githubusercontent.com/32765662/104777518-141df180-577c-11eb-970c-1f8d0466f6db.png)

 I'm not really sure what else to try.

Below I will list a sample of the contents of `Ct` and `idx` so that you don't have to dig into the code.

`%get Ct --from MATLAB
 print(Ct)`: https://pastebin.com/vtst448a

`%get idx --from MATLAB
 print(idx)`: `[ 5 28 53 98]`

**Comment by agahkarakuzu** [comment]
@emorevival looks like you are plotting data corresponding to a single entity, whereas the original code overlays 4 lines. 

You can achieve something like that using plotly figure's `add_trace` method. Not sure if `px.line` is a valid argument for this method, but still, I think it can give you an overall idea about what's happening here:

```python
import plotly.graph_objects as go
import numpy as np

fig = go.Figure()
for ii in [5,28,53,98]:
  fig.add_trace(go.Scatter(x=t, y=np.squeeze(np.asarray(np.reshape(Ct[:,ii],120))),mode='lines'))

fig.show()
```

Not sure if I get the syntax right above, but should be pretty much like this. Another thing I noticed is that you are passing a dataframe (`df`) that belongs to the example in your snippet. In plotly express, x and y arguments are to indicate which slice of the data encapsulated by dataframe `df` are going to be displayed. Whereas in your case you have `x` and `y` as freestanding variables, not in a dataframe.

**Comment by ddimoski** [comment]
Thanks for the pointers, we figured out what the problem was. Matlab's array indexing starts from 1, so the array of indices we were fetching from Matlab `idx` was giving us different results than expected in Python when we would call `Ct[:,idx]`. We added each line plot with `add_trace` as you suggested and removed the dataframe. 
![image](https://user-images.githubusercontent.com/20287217/105379361-0a2d4000-5c0d-11eb-892c-effd5776b722.png)
